### PR TITLE
Dev edward#448

### PR DESCRIPTION
**Pull request by JorisBassle**
Will push every once and a while for intermediate code review.
Don't delete the branch if PR is okay, I will continue with this one

### CODE REVIEW

File: src/store/fileBrowserStore.ts [hunk 1]
```diff
@@ -256,14 +258,16 @@ export const getchatsWithFiles = (received: boolean) => {
...
+// --> not used anywhere
+//
+// export const uploadFile = async (file: File, path = currentDirectory.value) => {
+//     const result = await Api.uploadFile(path, file);
+//     if ((result.status !== 200 && result.status !== 201) || !result.data)
+//         throw new Error('Could not create new folder');
+//
+//     currentDirectoryContent.value.push(createModel(result.data));
+//     await updateContent();
+// };
```
[the remark this answers is not in the document]
Comment (JonasWijne) [reply src/store/fileBrowserStore.ts:270]: commented code should be deleted

File: src/components/ContactAdd.vue [hunk 2]
```diff
@@ -1,86 +1,86 @@
 <template>
-    <div class="place-items-start">
-        <div class="grid grid-cols-2">
+    <div class='place-items-start'>
+        <div class='grid grid-cols-2'>
```
Comment (JonasWijne) [inline src/components/ContactAdd.vue:3]: other files use " , what is the standard in this repo ?

File: src/services/fileService.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+export const uploadTempFile = async (file: File): Promise<string> => {
+    const formData = new FormData();
+    formData.append('file', file);
+    let config: AxiosRequestConfig = {
+        headers: {
+            'Content-Type': 'multipart/form-data',
+        },
+    };
+    return (await axios.post(`${endpoint}/upload`, formData, config)).data;
+};
```
Comment (JonasWijne) [inline src/services/fileService.ts:15]: add a comment to the function which explains why we do this to stop any future confusion

File: src/store/authStore.ts [hunk 4]
```diff
@@ -21,14 +22,16 @@ const authState = reactive<AuthState>({
...
 
-export const myYggdrasilAddress = async () => {
-    const res = await axios.get(`${window.location.origin}/api/v2/locations/yggdrasil`);
-    return res.data;
-};
+const myLocation = ref<string>('');
+// export const myYggdrasilAddress = async () => {
+//     const res = await axios.get(`${window.location.origin}/api/v2/locations/yggdrasil`);
+//     return res.data;
+// };
```
Comment (JonasWijne) [inline src/store/authStore.ts:29]: For every line of code in comments a seal dies, this is true, many people know this
image

File: src/store/blockStore.ts [hunk 5]
```diff
@@ -1,25 +1,23 @@
...
-    }
-};
+// export const initBlocklist = async () => {
+//     try {
+//         const axiosResponse = await axios.get(`${config.baseUrl}api/v1/blocked`);
+//         blocklist.value = axiosResponse.data;
+//     } catch (e) {
+//         console.log('could not get blocklist');
+//     }
+// };
```
Comment (JonasWijne) [inline src/store/blockStore.ts:12]: image

File: src/store/blockStore.ts [hunk 6]
```diff
@@ -1,25 +1,23 @@
...
 export const isBlocked = (userId: string) => blocklist.value.includes(userId);
 
-export const deleteBlockedEntry = async (user: string) => {
-    await axios.delete(`${config.baseUrl}api/v1/blocked/${user}/`);
-    blocklist.value = blocklist.value.filter(x => x !== user);
-};
+// export const deleteBlockedEntry = async (user: string) => {
+//     await axios.delete(`${config.baseUrl}api/v1/blocked/${user}/`);
+//     blocklist.value = blocklist.value.filter(x => x !== user);
+// };
```
Comment (JonasWijne) [inline src/store/blockStore.ts:23]: image

File: src/store/fileBrowserStore.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (JonasWijne) [reply src/store/fileBrowserStore.ts:270]: image

File: src/store/fileBrowserStore.ts [hunk 7]
```diff
@@ -141,6 +142,7 @@ export const createDirectory = async (name: string, path = currentDirectory.valu
 export const uploadFiles = async (files: File[], path = currentDirectory.value) => {
     Promise.all(
         files.map(async (f): Promise<void> => {
+            // const id = await uploadTempFile(f);
```
Comment (JonasWijne) [inline src/store/fileBrowserStore.ts:145]: image

**Review by JonasWijne — CHANGES_REQUESTED** [review]

**Comment by JensMeire** [comment]
A lot of seals died in cuz of this PR

**Comment by GillesMommerency** [comment]
image

File: src/components/Dashboard/Post.vue [hunk 8]
```diff
@@ -453,7 +450,7 @@
     });
 
     const myAvatar = computed(() => {
-        return calcExternalResourceLink(`http://[${myLocation.value}]/api/v1/user/avatar/default`);
+        return calcExternalResourceLink(`http://[${user.location}]/api/v1/user/avatar/default`);
     });
```
Comment (JonasWijne) [inline src/components/Dashboard/Post.vue:454]: is this code still used ?

Comment (JonasWijne) [reply src/components/Dashboard/Post.vue:454]: why http

**Review by JonasWijne — CHANGES_REQUESTED** [review]

**Comment by JorisBassle** [comment]
@JonasWijne 
image

**Review by JonasWijne — APPROVED** [review]

**Pull request merged**